airbyte-cdk/python/unit_tests/sources/declarative/auth/test_oauth.py

```diff
@@ -66,6 +66,48 @@ def test_refresh_request_body(self):
         }
         assert body == expected
 
+    def test_refresh_with_encode_config_params(self):
```

Comment:
let's unit test the base64 in https://github.com/airbytehq/airbyte/blob/master/airbyte-cdk/python/unit_tests/sources/declarative/interpolation/test_filters.py

Reply:
Done